<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Εργαλεία Διαχειρηστή</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="adminPasswordChange">
     <property name="text">
      <string>Αλλαγή Κωδικού Διαχειριστή</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2" colspan="2">
    <widget class="QPushButton" name="addSingleItem">
     <property name="text">
      <string>Εισαγωγή Νέου Προϊόντος</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="changeProductList">
     <property name="toolTip">
      <string>Εισαγωγή νέου αρχείου λίστας προϊόντων</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>Αλλαγή Λίστας Προϊόντων</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
